### DISCUSSION TITLE
Can 2 pnp search filter web parts be connected to 1 search results web part?

### DISCUSSION THREAD

**Discussion by webbrewer** [microsoft-search/pnp-modern-search#3878] [General]
It only seems to allow for one filter connection but it would be very useful if we could add more .... I know you can add multiple filters in one web part but my use case requires repositioning other filters on the page. 
Also, additional filters override the first filter, so you can't refine the results the way ecommerce sites like Amazon do.

**Comment by kasperbolarsen** [comment]
Why? Can you describe the scenarios where this will be an advantage

**Reply by webbrewer** [reply]
My design team wants to separate the filters: _"I know you can add multiple filters in one web part but my use case requires repositioning other filters on the page."_

**Reply by kasperbolarsen** [reply]
Unless we see additional requests about this, I think you will have to ask your own developers to implement this. 
We welcome a PR if you make one.

**Reply by wobba** [reply]
Answer is no. If you add all filters you want in one webpart, then if you select one value, all others will be refined to show applicable values according to the first filter, reducing the options.

**Comment by webbrewer** [comment]
Thanks!
